Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/preferences/PreferencesFragment.java

```diff
@@ -197,6 +204,36 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
             } else {
                 pushNotificationManager.removePushDeviceUsingStoredToken();
             }
+        } else if (key.equals("language")) {
+            LanguageHelper languageHelper = new LanguageHelper(sharedPreferences.getString(key,"en"));
+            Locale.setDefault(languageHelper.getLocale());
+            Configuration configuration = new Configuration();
+
+            if (android.os.Build.VERSION.SDK_INT <= Build.VERSION_CODES.JELLY_BEAN){
+                configuration.locale = languageHelper.getLocale();
+            } else {
+                configuration.setLocale(languageHelper.getLocale());
+            }
+
+            getActivity().getResources().updateConfiguration(configuration,
+                    getActivity().getResources().getDisplayMetrics());
```

Comment:
The content api is the REST api call that provides all the content for the app (p.e. gear, pets, mounts. etc.). It can be synced with `apiClient.apiService.getContent()` when you have an apiClient object. Like it is done [here](https://github.com/HabitRPG/habitrpg-android/blob/develop/Habitica/src/main/java/com/habitrpg/android/habitica/ui/fragments/inventory/customization/AvatarOverviewFragment.java#L64)